openhub/affiliated_committers.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from community.git import get_owner
+from openhub.models import MostCommit, MostRecentCommit, AffiliatedCommitter
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def get_affiliated_committers_data(json_object):
+    data = json_object['response']['result'
+                                   ]['affiliated_committers']['affiliator']
+    return data
+
+
+def import_data(affiliator):
+    org = get_owner()
+    name = affiliator.get('name', None)
+    kudos = affiliator.get('kudos', None)
+    level = affiliator.get('level', None)
+    try:
+        (cr1, create1) = MostCommit.objects.get_or_create(
+            **affiliator['most_commits']
+            )
+        if create1:
+            cr1.save()
+        (cr2, create2) = MostRecentCommit.objects.get_or_create(
+            **affiliator['most_recent_commit']
+            )
+        if create2:
+            cr2.save()
+        (c, created) = AffiliatedCommitter.objects.get_or_create(
+            name=name,
```

Comment:
use `**`, but you need to modify `affiliator` first.  e.g. `affiliator['most_commits'] = cr1`

Reply:
I was not able to use `**` here because `org` field is not into `dict`